EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -936,7 +936,10 @@ public void StoreSystemNote(HistoryEntry he, string txt, bool send = false)
                 // Also notify various "quick" listeners
                 if (he.UpdateSystemNote(txt, send))
                 {
-                    _uncommittedNoteHistoryEntry = he;
+                    if (_uncommittedNoteHistoryEntry == null)
```

Comment:
I guess all of the conditions that would cause a new note to be sent would first cause the text box to be unfocussed and therefore cause the old note to be sent.